Library/Homebrew/cask/lib/hbc/system_command.rb

```diff
@@ -81,31 +81,16 @@ def each_output_line(&b)
 
       write_input_to(raw_stdin)
       raw_stdin.close_write
-      each_line_from [raw_stdout, raw_stderr], &b
 
+      ::Utils::IOSelector
+        .each_line_from(stdout: raw_stdout, stderr: raw_stderr, &b)
```

Comment:
Yes, I figure that should be `&block` too; however that should be left to another PR.

I have left `&b` untouched because the corresponding method signature does not really relate to this issue. I’d prefer to keep commits focused, and diffs too ☺️